src/codegen/codegen_c.cc

```diff
@@ -519,14 +519,22 @@ void CodeGenC::VisitExpr_(const Not *op, std::ostream& os) {  // NOLINT(*)
 void CodeGenC::VisitExpr_(const Call *op, std::ostream& os) {  // NOLINT(*)
   if (op->call_type == Call::Extern ||
       op->call_type == Call::PureExtern) {
-    os << op->name << "(";
-    for (size_t i = 0; i < op->args.size(); i++) {
-      this->PrintExpr(op->args[i], os);
-      if (i < op->args.size() - 1) {
-        os << ", ";
+    if (op->name == "is_nan") {
```

Comment:
Let us add the intrinsic to https://github.com/dmlc/tvm/blob/master/include/tvm/ir.h#L1324 and use is_intrinsic function as in the following parts

Reply:
consider to make the string consistent(isnan instead of is_nan)